packages/icons/src/components/iconSizes.ts

```diff
@@ -0,0 +1,7 @@
+export const iconSizes = {
```

Comment:
Every component will have their different sizes, fox example cryptoIcon will have only 24, 42. Flag will have only one size probably etc. I would prefer to place this to component files directly because sizes could be different also on web later.